dialogTemplate/fr.json

```diff
@@ -0,0 +1,806 @@
+				"mets les lumières de {partout:=>Room} en {va se coucher:=>Scene}",
+				"mets les lumières {partout:=>Room} en {va se coucher:=>Scene}",
+				"mets les lumières dans la {chambre:=>Room} en {va se coucher:=>Scene}",
+				"mets la lumière de la {chambre:=>Room} en {va se coucher:=>Scene} s'il te plait",
+				"mets la lumière du {bureau:=>Room} en {va se coucher:=>Scene} s'il te plait",
+				"mets la lumière dans le {bureau:=>Room} en {va se coucher:=>Scene} s'il te plait",
+				"mets la lumière de {partout:=>Room} en {va se coucher:=>Scene} s'il te plait",
+				"mets la lumière {partout:=>Room} en {va se coucher:=>Scene} s'il te plait",
+				"mets la lumière dans la {chambre:=>Room} en {va se coucher:=>Scene} s'il te plait",
+				"mets la lumière de la {chambre:=>Room} en {va se coucher:=>Scene}",
+				"mets la lumière du {bureau:=>Room} en {va se coucher:=>Scene}",
+				"mets la lumière dans le {bureau:=>Room} en {va se coucher:=>Scene}",
+				"mets la lumière de {partout:=>Room} en {va se coucher:=>Scene}",
+				"mets la lumière {partout:=>Room} en {va se coucher:=>Scene}",
+				"mets la lumière dans la {chambre:=>Room} en {va se coucher:=>Scene}",
+				"mets l'éclairage de la {chambre:=>Room} en {va se coucher:=>Scene} s'il te plait",
+				"mets l'éclairage du {bureau:=>Room} en {va se coucher:=>Scene} s'il te plait",
+				"mets l'éclairage dans le {bureau:=>Room} en {va se coucher:=>Scene} s'il te plait",
+				"mets l'éclairage de {partout:=>Room} en {va se coucher:=>Scene} s'il te plait",
+				"mets l'éclairage {partout:=>Room} en {va se coucher:=>Scene} s'il te plait",
+				"mets l'éclairage dans la {chambre:=>Room} en {va se coucher:=>Scene} s'il te plait",
+				"mets l'éclairage de la {chambre:=>Room} en {va se coucher:=>Scene}",
+				"mets l'éclairage du {bureau:=>Room} en {va se coucher:=>Scene}",
+				"mets l'éclairage dans le {bureau:=>Room} en {va se coucher:=>Scene}",
+				"mets l'éclairage de {partout:=>Room} en {va se coucher:=>Scene}",
+				"mets l'éclairage {partout:=>Room} en {va se coucher:=>Scene}",
+				"mets l'éclairage dans la {chambre:=>Room} en {va se coucher:=>Scene}",
+				"mets les lumières en {va se coucher:=>Scene} s'il te plait",
+				"mets les lumières en {va se coucher:=>Scene} s'il te plait",
+				"mets les lumières en {va se coucher:=>Scene} s'il te plait",
+				"mets les lumières en {va se coucher:=>Scene} s'il te plait",
+				"mets les lumières en {va se coucher:=>Scene} s'il te plait",
+				"mets les lumières en {va se coucher:=>Scene}",
+				"mets les lumières en {va se coucher:=>Scene}",
+				"mets les lumières en {va se coucher:=>Scene}",
+				"mets les lumières en {va se coucher:=>Scene}",
+				"mets les lumières en {va se coucher:=>Scene}",
+				"mets la lumière en {va se coucher:=>Scene} s'il te plait",
+				"mets la lumière en {va se coucher:=>Scene} s'il te plait",
+				"mets la lumière en {va se coucher:=>Scene} s'il te plait",
+				"mets la lumière en {va se coucher:=>Scene} s'il te plait",
```

Comment:
Cannot have multiple times the same utterance. This cannot pass distribution. Check projectalice-sk check routine: https://docs.projectalice.io/skill-development/publishing-to-skill-store.html#checking-your-skill